cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -181,6 +181,21 @@ describe('Top toolbar tests.', function() {
 			.should('have.attr', 'align', 'justify');
 	});
 
+	it('Apply spacing', function() {
+		cy.get('#tb_editbar_item_linespacing')
+			.click();
+
+		helper.reselectAllText();
```

Comment:
Why is this selectAllText() here? We need text selection if we would like to apply some formatting on the text, which is done before this method call, or when we would like to check the selected text's properties, which is done later in this test code.